commands/template/template.go

```diff
@@ -626,7 +626,7 @@ func (t *Templating) Run() (err error) {
 	tempDir, err := ioutil.TempDir(t.gitDir, "butler")
 	if err != nil {
 		err = errors.Wrap(err, "create temp folder failed")
-		return
+		return err
```

Comment:
The error is set via named return values

Reply:
And you think the bare return is better than the explicit return?

Reply:
in this case, it's manageable but in general, I prefer the explicit return. I'm +1 with this change.